public/index.html

```diff
@@ -4,11 +4,23 @@
   <link rel="stylesheet" type="text/css" href="styles/interface.css">
   <script src="/socket.io/socket.io.js"></script>
   <script src="scripts/main.js"></script>
+  <script src="http://code.jquery.com/jquery-1.11.0.min.js"></script>
```

Comment:
do y'all wanna use jQuery then? I'm down either way, though I'm not convinced that we need anything in jQuery yet that we can't fairly easily do with plain ole JS. Especially with the neato `document.querySelector`